best-practices/README.md

```diff
@@ -211,8 +211,11 @@ JavaScript
   HTML elements. #462
 * Avoid targeting HTML elements using classes intended for styling
   purposes. #462
+* Use [Prettier] to ensure consistent formatting across the codebase. Run
+  Prettier during CI and, if any files would be changed, fail the build.
```

Comment:
Should `would` be `should`? (say that 10 times fast! 🙃)

Or, perhaps I’m not aware of how Prettier works…does it change the files?

Reply:
How much wood should a wood should would if a wood should would should wood?

Reply:
@tysongach Prettier completely reformats your files.

Reply:
Got it. Perhaps a slight rewording to something like this would be more clear?

```diff
- Run Prettier during CI and, if any files would be changed, fail the build.
+ Run Prettier during CI and fail the build if it finds any files that need to be changed.
```

Reply:
The `-l` option to `prettier` lists all files that _would_ be changed if you ran `prettier --fix`. It's like a dry-run, and fails if there are any files listed. I'm guessing that the intent of the wording is that it's the responsibility of contributors to run `prettier --fix` (or configure prettier for their editor), and if prettier hasn't been run then CI will fail, instead of automatically trying to fix formatting for you.

Reply:
@sharplet yes, that's exactly the intent. I personally have my editor wired up to auto-format on save, but if a developer new to the project hasn't done that yet, I'd want CI to catch it and disallow merging until we help the new developer configure their editor (and then fix all the files)

Reply:
Makes sense! Thanks all, for clarifying.